releases/stellar/2025-02-ITS-v1.1.0.md

```diff
@@ -0,0 +1,42 @@
+# Stellar ITS v1.1.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @nbayindirli (<noah@interoplabs.io>)       |
+| **Deployment** | @RiceAndMeet (<steven@interoplabs.io>) |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Crates](https://crates.io/crates/stellar-interchain-token-service/1.1.0)
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases/tag/stellar-interchain-token-service-v1.1.0)
+
+## Background
+
+- This is the v1.1.0 Stellar ITS release.
+
+Contract changes in the release:
+
+1. Use contractstorage macro on storage enum.
```

Comment:
Link to the other changelog instead of duplicating, and mention user facing ITS specific changes otherwise